docs/installation/install.md

```diff
@@ -135,6 +135,41 @@ In a nutshell, after having installed the CLI, you have to set up your identity:
 aws configure
 ```
 
+The minimum permissions required by the user (in addition to the ones required to create and operate the EKS cluster) are the following:
```

Comment:
This is not clear to me. If the user must already have the permissions to create and operate the EKS cluster, what are those permissions needed?
Probably a more verbose explanation would help here.

Reply:
I spent a few hours understanding how eks manages to integrate iam users with k8s users. The IAM user is only used to authenticate with the k8s cluster, and the cluster uses that authentication to remap the IAM user to a k8s user. This means that the management of permissions to "operate" on the cluster is not managed via IAM, but via k8s. So what I have written in parentheses loses its meanings.

Reply:
If an admin were to create a cluster and then delegate the liqo installation to another user (who has no create or destroy permissions on EKS, CloudFormation and EC2) the only additional permission needed would be **DescribeVPC**

Reply:
This last comment looks interesting to me. It is worth mentioning in the documentation?

Reply:
I explained badly. There are 2 possible scenarios.
- A user who has the permissions to create and manage EKS clusters.
- A user who does not have permissions to create or manage EKS, but wants to install Liqo.
In both cases the permissions required to install Liqo are the ones that I'm adding to the documentation.

Reply:
I would add something like `Even if you are not an EKS administrator you can install Liqo`.